README.md

```diff
@@ -1,14 +1,16 @@
-# TrackJS JavaScript Integrations
-
-This repository has several live code examples for various JavaScript integrations with TrackJS.  Each sub-folder has a descriptive README with more information.
-
-### Build Integrations
-* [Basic](./basic/) - Simplest example possible.
-* [AMD and RequireJS](./amd-loading-with-require) - Integrate TrackJS with a RequireJS/AMD workflow
-* [Bundling with Gulp](./bundling-with-gulp) - Adding TrackJS to a minified script bundle using Gulp
-* [Webpack and NPM](./webpack-and-npm) - Uses exports-loader to make loading TrackJS via NPM seamless
-* [Browserify](./browserify-and-npm) - Bundling TrackJS with your app bundle using Browserify
-* [Bower](./bower-and-gulp) - Uses Gulp to bundle TrackJS, other bower components and your application
-* [JSPM and Babel](./jspm-with-babel) - Shows how to use TrackJS with UMD and ES6 
-* [TypeScript](./typescript) - Use TrackJS TypeScript definitions
- 
\ No newline at end of file
+<p align="center">
+  <a href="https://trackjs.com/" target="_blank" align="center">
+    <img src="https://trackjs.com/assets/images/brand/logo_full_charcoal_red.svg" width="280">
+  </a>
+  <br />
+</p>
+
+# TrackJS Usage and Integration Examples
+
+This repository contains example projects for using and integrating the TrackJS Error Monitoring agents with common JavaScript patterns and frameworks.
```

Comment:
Maybe link to express and nextjs examples here like the old one did?

Reply:
Sure.